FitNesseRoot/DbFit/AcceptanceTests/JavaTests/InformixTests/FlowMode/InformixMode/SetUp/content.txt

```diff
@@ -0,0 +1,3 @@
+!|dbfit.InformixTest|
+
+|Connect|localhost:16466|dbfit|DBFIT|dbfit_ifx|
```

Comment:
Yes, we have a mix. Actually - just a few have been converted to `ConnectUsingFile` (ones for which there has been some demand). Yes, we can handle it in separate PR.